modules/ble/src/main/java/com/gluonhq/attach/ble/BleService.java

```diff
@@ -216,7 +216,7 @@
     void stopScanning();
 
     /**
-     * Configure the current iOS device as a Bluetooth beacon, and start
+     * Configure the current device as a Bluetooth beacon, and start
```

Comment:
Maybe rename to `DalvikBleService`? (Though it will require a change in Substrate)

Reply:
done